nuget/framework/nunit.nuspec

```diff
@@ -30,7 +30,6 @@ Supported platforms:
     <dependencies>
       <group targetFramework="netstandard1.6">
         <dependency id="NETStandard.Library" version="1.6.0" />
-        <dependency id="Microsoft.NETCore.Portable.Compatibility" version="1.0.1" />
```

Comment:
I confirmed that the Android and iOS are loading their versions of the framework, so I don't know why they are pulling in the .NET Standard dependencies, but they are. I even tested by removing the .NET Standard target and leaving these dependencies and they still get pulled in. It is this dependency that is causing the problems and it was trivial to remove it.